### PR TITLE
Tl/driver and die

### PR DESCRIPTION
**Pull request by TengLuo**
src/driver/driver.py
Create a driver for sub-system, Q/A, die and player token completed, board function left

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,11 @@
-numpy==1.19.0
```
Comment (johnChandlerr) [inline requirements.txt:1]: I'm not sure about the numpy requirement, but I do remember Matt's question_manager.py imported and used pandas.

Comment (TengLuo) [reply requirements.txt:1]: Looks like after pip freeze > requirements.txt stored currently installed packages, I will revert back

**Comment by TengLuo** [comment]
The driver will open the board GUI for board function demo

**Comment by johnChandlerr** [comment]
Looks good Teng, good job.

**Pull request merged**